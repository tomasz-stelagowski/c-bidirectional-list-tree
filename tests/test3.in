ADD_NODE 0
ADD_NODE 0
ADD_NODE 2
ADD_NODE 0
ADD_NODE 3
ADD_NODE 3
RIGHTMOST_CHILD 3
ADD_NODE 3
DELETE_NODE 5
RIGHTMOST_CHILD 6
DELETE_SUBTREE 4
ADD_NODE 3
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 2
ADD_NODE 8
DELETE_NODE 7
RIGHTMOST_CHILD 8
DELETE_NODE 8
ADD_NODE 1
ADD_NODE 2
RIGHTMOST_CHILD 3
ADD_NODE 0
DELETE_NODE 6
RIGHTMOST_CHILD 10
ADD_NODE 2
RIGHTMOST_CHILD 1
RIGHTMOST_CHILD 13
ADD_NODE 11
DELETE_NODE 1
RIGHTMOST_CHILD 11
DELETE_SUBTREE 13
ADD_NODE 11
RIGHTMOST_CHILD 0
ADD_NODE 14
DELETE_NODE 2
RIGHTMOST_CHILD 0
DELETE_NODE 11
DELETE_NODE 12
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 3
DELETE_NODE 14
DELETE_NODE 15
RIGHTMOST_CHILD 0
DELETE_NODE 16
RIGHTMOST_CHILD 0
DELETE_NODE 3
RIGHTMOST_CHILD 0
DELETE_NODE 9
RIGHTMOST_CHILD 0
