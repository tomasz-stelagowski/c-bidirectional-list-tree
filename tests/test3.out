OK
OK
OK
OK
OK
OK
6
OK
OK
-1
OK
OK
2
3
OK
OK
9
OK
OK
OK
9
OK
OK
-1
OK
10
-1
OK
OK
14
OK
OK
12
OK
OK
12
OK
OK
15
9
OK
OK
16
OK
3
OK
9
OK
10
